### PR TITLE
Elastic Search - "missing features or docs" #60

### PR DESCRIPTION
**Pull request by devypt**
@cebe  This not finished yet, but it's good to have your review on it.

### CODE REVIEW

**Comment by cebe** [comment]
@beowulfenator can you take a look at this one?

File: docs/guide/mapping-indexing.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Class BookIndex extends yii\elasticsearch\ActiveRecord;
+
+     /**
+     * sets up the index for this record
+     * @param Command $command
+     */
+    public static function setUpMapping()
+    {
+        $db = $this->getDb();
```
Comment (cebe) [inline docs/guide/mapping-indexing.md]: should be `static::getDb()`.

File: docs/guide/mapping-indexing.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                    'status' => ["type" => "long"],
+                 
+                ]
+            ]
+        ]);
+    }             
+
+```
+Later on you just need to call this method any time you want to apply the new mapping.
+
```
Comment (cebe) [inline docs/guide/mapping-indexing.md]: need information on the mapping changes. `deleteMapping` call above is dangerous.

**Comment by beowulfenator** [comment]
I've updated this PR here:
https://github.com/yiisoft/yii2-elasticsearch/pull/81

**Pull request closed without merging**